### PR TITLE
Panel: Use Tabs in panel inspector

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:
Replacing the `<Select />` in the panel inspector with tabs. Adding a super simple `<TabContent />` component to wrap whatever we render. 

This does not handle tabs for navigation within TabContent. I think we can handle that in a separate PR as it requires some extra work on the `<TabsBar />`.

### CODE REVIEW

File: public/app/features/dashboard/components/Inspector/PanelInspector.tsx [hunk 1]
```diff
@@ -139,9 +139,8 @@ export class PanelInspector extends PureComponent<Props, State> {
             />
           </div>
         )}
-        <div style={{ border: '1px solid #666' }}>
```
Comment (peterholmberg) [inline public/app/features/dashboard/components/Inspector/PanelInspector.tsx]: Removed this border as it was not theme agnostic and I could not really see the value of it now. If anyone has strong feelings about keeping it I don't mind adding it with correct colors.

**Review by hugohaggmark — APPROVED** [review]
LGTM!

File: public/app/features/dashboard/components/Inspector/PanelInspector.tsx (hunk 1 above)
Comment (ryantxu) [reply public/app/features/dashboard/components/Inspector/PanelInspector.tsx]: perfect -- that was me... hack hack hack.

I put all the pretend tab stuff there waiting for it to get replaced :)

**Review by ryantxu — APPROVED** [review]
me too!

**Review by mckn — APPROVED** [review]
Super! 🙌

File: packages/grafana-ui/src/components/Tabs/Tabs.story.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { Tab } from './Tab';
+import { TabContent } from './TabContent';
+
+export default {
+  title: 'UI/Tabs/TabsExample',
+  decorators: [withCenteredStory],
+};
+
+const tabs = [
+  { label: '1st child', key: 'first', hide: false, active: true },
```
Comment (dprokop) [inline packages/grafana-ui/src/components/Tabs/Tabs.story.tsx]: Can we remove unused `hide` properties?

File: packages/grafana-ui/src/components/Tabs/styles.ts [hunk 3]
```diff
@@ -7,6 +7,9 @@ export const getTabsStyle = stylesFactory((theme: GrafanaTheme) => {
   const tabBorderColor = selectThemeVariant({ dark: colors.dark9, light: colors.gray5 }, theme.type);
 
   return {
```
Comment (dprokop) [inline packages/grafana-ui/src/components/Tabs/styles.ts]: Ref https://github.com/grafana/grafana/pull/21328#issuecomment-573015290 -> can we move the styles to the component files?

**Review by dprokop — APPROVED** [review]
Some nits, apart from that LGTM!

**Pull request merged**